examples/05_advanced/plot_ica_neurovault.py

```diff
@@ -0,0 +1,146 @@
+"""
+NeuroVault cross-study ICA maps.
+================================
+
+This example shows how to download statistical maps from
+NeuroVault, label them with NeuroSynth terms,
+and compute ICA components across all the maps.
+
+See :func:`nilearn.datasets.fetch_neurovault`
+documentation for more details.
+
+"""
+# Author: Ben Cipollini
+# License: BSD
+# Ported from code authored by Chris Filo Gorgolewski, Gael Varoquaux
+# https://github.com/NeuroVault/neurovault_analysis
+import warnings
+
+import numpy as np
+from scipy import stats
+from sklearn.decomposition import FastICA
+
+from nilearn.datasets import fetch_neurovault_filtered
```

Comment:
I think that I prefer the name "fetch_neurovault": what "_filtered" means isn't very clear unless you read the docstring.

Other option: "fetch_neurovault_query"?

Anyhow, you need to update docs/module/references.rst, which points to the old name (and misses a link to the second function), hence leads to failures in circleCI.